### ISSUE TITLE
improve ability to disable ipv6

### ISSUE THREAD

**Issue by anderbubble** [hpcng/warewulf#976]
### Summary

```
IPV6INIT=yes
IPV6_AUTOCONF=yes
IPV6_DEFROUTE=yes
```

These are configured statically. We should be able to disable them.

### General information

- [X] I have searched the issues of this repo and believe this is not a duplicate

**Comment by mslacken** [comment]
No, although a static IPvv6 is configured the autoconfiguration is sensible as this will also create the IPv6 address based on the MAC address and the dynamic privacy addresses which are used for the outgoing traffic.

**Comment by anderbubble** [comment]
If I recall correctly (I'm transferring this in from personal notes) I think I had a use case where the presence of the IPv6 interface was interfering with something / causing the node to attempt to communicate via IPv6 rather than IPv4 when IPv6 wasn't actually available. I wanted to be able to disable IPv6 entirely but, as we discussed in the last community meeting, this was difficult to do without losing the rest of the wwinit functionality.

Perhaps my concern here would go away if we split wwinit into multiple smaller overlays as you mentioned.
